src/constants/constants.ts

```diff
@@ -16,6 +16,7 @@ export const getDefaultSlippageDecimalPercentageForSwapper = (
   switch (swapperName) {
     case SwapperName.Zrx:
     case SwapperName.OneInch:
+    case SwapperName.ArbitrumBridge:
```

Comment:
If there is indeed no slippage for arbitrum bridge (which I believe is the case), we can use `DEFAULT_ARBITRUM_BRIDGE_SLIPPAGE` of `0` and use accordingly in swapper instead of hardcode inline `0`

Reply:
69ddc11737